core/src/main/java/io/fabric8/maven/core/openshift/BuildService.java

```diff
@@ -63,7 +63,7 @@ public Build startBuild(OpenShiftClient client, File dockerTar, String buildName
             }
             if (exp.getCause() instanceof IOException && exp.getCause().getMessage().contains("Stream Closed")) {
                 log.error("Build for %s failed: %s", buildName, exp.getCause().getMessage());
-                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is')");
+                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is -n openshift')");
```

Comment:
It's better than pointing the user to the current namespace, because the FIS imagestreams are installed in the "openshift" namespace, and it is the default in f-m-p (rh branch). I'll check if I can improve the message.